docs_src/microservices/core/command/Ch-Command.md

```diff
@@ -23,53 +23,52 @@ way to simplify communications with the devices. There are two types of commands
 - a GET command requests data from the device.  This is often used to request the latest sensor reading from the device.  
 - PUT commands request to take action or [actuate](../../../general/Definitions.md#actuate) the device or to set some configuration on the device.
 
-In most cases, GET commands are simple requests for the latest sensor reading from the device.  Therefore, the request is often parameter-less (requiring no parameters or body in the request).  As of the Geneva release, GET requests do allow query parameters to be added to the GET command requests (i.e. `/api/v1/device/{deviceID}/command/{commandID}?param1=value&param2=value`).  PUT commands often require a request body where the body provides a key/value pair array of values used as parameters in the request (i.e. `{"additionalProp1": "string", "additionalProp2": "string"}`).
+In most cases, GET commands are simple requests for the latest sensor reading from the device.  Therefore, the request is often parameter-less (requiring no parameters or body in the request).  PUT commands require a request body where the body provides a key/value pair array of values used as parameters in the request (i.e. `{"additionalProp1": "string", "additionalProp2": "string"}`).
```

Comment:
In the profiles we changed PUT commands to SET commands.  I think we should still to that terminology here.